CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Added a validation to **validate**, checking that each script, integration and playbook have a README file. This validation only runs when the command is called with either the `-i` or the `-g` flag.
 * Fixed a regression issue with **doc-review**, where  the `-g` flag did not work
 * Improved the detection of errors in **doc-review** command.
+* Fixed an issue in **validate**, where README is not empty, is only checked in packs that contain playbooks, or written by partners. 
```

Comment:
```suggestion
* Fixed an issue in **validate**, where non-empty README files were only checked in packs that contain playbooks, or written by partners. 
```

Reply:
@dorschw don't think this is the best wording. please look again and confirm if ok by you.